src/cpp/src/llm_pipeline.cpp

```diff
@@ -9,55 +9,20 @@
 #include <openvino/openvino.hpp>
 #include "openvino/genai/generation_config.hpp"
 #include "openvino/genai/llm_pipeline.hpp"
+#include "llm_pipeline_base.hpp"
+#include "llm_pipeline_static.hpp"
 #include "utils.hpp"
 #include "text_callback_streamer.hpp"
 
-namespace {
-
-const std::string STREAMER_ARG_NAME = "streamer";
-const std::string CONFIG_ARG_NAME = "generation_config";
-
-ov::genai::GenerationConfig from_config_json_if_exists(const std::filesystem::path& model_path) {
-    auto config_file_path = model_path / "generation_config.json";
-    if (std::filesystem::exists(config_file_path)) {
-        return ov::genai::GenerationConfig((config_file_path).string());
-    } else {
-        return ov::genai::GenerationConfig{};
-    }
-}
-
-ov::genai::StreamerVariant get_streamer_from_map(const ov::AnyMap& config_map) {
-    ov::genai::StreamerVariant streamer = std::monostate();
-
-    if (config_map.count(STREAMER_ARG_NAME)) {
-        auto any_val = config_map.at(STREAMER_ARG_NAME);
-        if (any_val.is<std::shared_ptr<ov::genai::StreamerBase>>()) {
-            streamer = any_val.as<std::shared_ptr<ov::genai::StreamerBase>>();
-        } else if (any_val.is<std::function<bool(std::string)>>()) {
-            streamer = any_val.as<std::function<bool(std::string)>>();
-        }
-    }
-    return streamer;
-}
-
-ov::genai::OptionalGenerationConfig get_config_from_map(const ov::AnyMap& config_map) {
-    if (config_map.count(CONFIG_ARG_NAME))
-        return config_map.at(CONFIG_ARG_NAME).as<ov::genai::GenerationConfig>();
-    else
-        return std::nullopt;
-}
-
-}
-
 namespace ov {
 namespace genai {
 
 ov::genai::EncodedResults greedy_decoding(
-    ov::InferRequest& model_runner, 
-    ov::Tensor prompts, 
-    ov::Tensor attention_mask, 
-    const GenerationConfig sampling_params, 
-    const std::shared_ptr<StreamerBase> streamer, 
```

Comment:
Just for your future contributions (don't revert in this PR). If you want to fix trailing white-spaces, you should do it everywhere and add a check that no new spaces are added to precommit.

Inplace fixes increase the diff size. At the same time there's nothing preventing the same person who originally added white-spaces to add them again. Which would increase the diff size for that person's PR as well.

Reply:
Wasn't intentional, just my editor :)